network/network.go

```diff
@@ -369,9 +384,11 @@ func (n *network) GetAcceptedFrontier(validatorIDs ids.ShortSet, chainID ids.ID,
 				chainID,
 				requestID)
 			n.getAcceptedFrontier.numFailed.Inc()
+			n.sendFailRateCalculator.Observe(1, n.clock.Time())
```

Comment:
Can we only call `n.clock.Time()` once in these functions? (Calling `.Time` takes time, but also calling `.Observe` with the same time is much faster.